src/NeoPixelBus.h

```diff
@@ -71,6 +71,7 @@ License along with NeoPixel.  If not, see
 
 #elif defined(ARDUINO_ARCH_ESP32)
 
+#include "internal/NeoEspRmtMethod.h"
```

Comment:
Only common files between esp8266 and esp32 are named just "esp".  This should be NeoEsp32RmtMethod.h